src/NHibernate/Criterion/Projections.cs

```diff
@@ -214,26 +214,21 @@ public static AggregateProjection Sum(IProjection projection)
 		}
 
 		/// <summary>
-		/// A SQL projection, a typed select clause fragment
+		/// A SQL projection, a typed select clause fragment.
+		/// The string {alias} will be replaced by the alias of the root entity.
+		/// Criteria aliases can also be used: "{a}.Value + {bc}.Value". Such aliases need to be registered via call to AddCriteriaAliases("a", "bc")
 		/// </summary>
-		/// <param name="sql"></param>
-		/// <param name="columnAliases"></param>
-		/// <param name="types"></param>
-		/// <returns></returns>
-		public static IProjection SqlProjection(string sql, string[] columnAliases, IType[] types)
+		public static SQLProjection SqlProjection(string sql, string[] columnAliases, IType[] types)
```

Comment:
Same here.

Reply:
Deprecate and add `Projections.Sql` instead?

Reply:
Ok. Maybe for consistency do the same with `SqlGroupProjection` and `GroupProjection`.

Reply:
Well, it may conflict with a `Group` method, but it seems to me to be a bit the same thing. I do not really make sense of these variations here...

Reply:
Done ( I didn't touch `GroupProjection`)